test/phpunit/csvImportValidator/CsvAccessionTitleLengthTest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+use org\bovigo\vfs\vfsStream;
+
+/**
+ * @internal
+ *
+ * @covers \CsvFieldLengthValidator
+ */
+class CsvAccessionTitleLengthTest extends \PHPUnit\Framework\TestCase
+{
+    protected $vdbcon;
+    protected $context;
+    protected $vfs;
+    protected $csvData;
+    protected $csvInvalidData;
+    protected $csvHeader;
+
+    public function setUp(): void
+    {
+        $this->context = sfContext::getInstance();
+        $this->vdbcon = $this->createMock(PropelPDO::class);
+
+        $this->csvHeader = 'accessionNumber,acquisitionDate,sourceOfAcquisition,title,culture';
+
+        $this->csvData = [
+            '"","","","Valid Title","en"',
+            '"","","","Another Valid Title","en"',
+        ];
+
+        $this->csvInvalidData = [
+            '"","","","Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis s","en"',
+        ];
+
+        // define virtual file system
+        $directory = [
+            'unix_csv_valid.csv' => $this->csvHeader."\n".implode("\n", $this->csvData),
+            'unix_csv_invalid.csv' => $this->csvHeader."\n".implode("\n", $this->csvInvalidData),
+        ];
+
+        $this->vfs = vfsStream::setup('root', null, $directory);
+    }
+
+    /**
+     * @dataProvider csvValidatorTestProvider
+     *
+     * Generic test - options and expected results from csvValidatorTestProvider()
+     *
+     * @param mixed $options
+     */
+    public function testCsvValidator($options)
+    {
+        $filename = $this->vfs->url().$options['filename'];
+        $validatorOptions = isset($options['validatorOptions']) ? $options['validatorOptions'] : null;
+
```

Comment:
We should set the class name for QubitAccession here like:
`$validatorOptions['className'] = 'QubitAccession';`